<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWJL</class>
 <widget class="QWidget" name="QWJL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="button_deletePolygon">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>200</width>
     <height>500</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="widget_layer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>198</width>
      <height>498</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>400</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_addPolygon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_flipV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>flipV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_flipH">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>flipH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_inner">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>in+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>clip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>565</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
